lua/wire/server/wirelib.lua

```diff
@@ -578,28 +577,18 @@ function WireLib.Link_Start(idx, ent, pos, iname, material, color, width)
 
 	local input = ent.Inputs[iname]
```

Comment:
Wiremod has used this name for an incredibly long time, and if we were to change it now we'd at the very least have to have a conversation about preserving backwards compatibility. This is a discussion for sometime and someplace else.